src/FamilyHubs.SharedKernel.Razor/FullPages/SingleAutocomplete/ISingleAutocompletePageModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using FamilyHubs.SharedKernel.Razor.ErrorNext;
+using Microsoft.AspNetCore.Html;
+
+namespace FamilyHubs.SharedKernel.Razor.FullPages.SingleAutocomplete;
+
+//public enum AutocompleteMode
+//{
+//    Alphabetical,
+//    MatchStart
+//}
+
+public interface ISingleAutocompletePageModel
+{
+    string? ContentTop => null;
+    string? ContentBottom => null;
+    IReadOnlyDictionary<string, HtmlString>? ContentSubstitutions => null;
```

Comment:
question: when using collections, would it be better to use empty collections rather than null so we don't have to check for both null & empty? Appreciate there are cases when it makes sense to have the distinction between null and empty for collections, but does that apply here? Also unsure what the precedent is in the codebase, so happy to go with this if we're doing it elsewhere.

Reply:
Yeah, it's usually better to use an empty collection, but Dictionary doesn't support an empty static singleton object out of the box, like Enumerable.Empty<> or Array.Empty<>, so it would mean creating one, which seems a little too much for an interface (we wouldn't want to create an instance each time as it would lead to unnecessary memory churn).